Homework_1/Task_2.py

```diff
@@ -0,0 +1,22 @@
+#Треугольник существует только тогда, когда сумма любых двух его сторон больше третьей.
+#Дано a, b, c - стороны предполагаемого треугольника. 
+#Требуется сравнить длину каждого отрезка-стороны с суммой двух других.
+#Если хотя бы в одном случае отрезок окажется больше суммы двух других, 
+#то треугольника с такими сторонами не существует. 
+#Отдельно сообщить является ли треугольник разносторонним, равнобедренным или равносторонним.
+
+def is_valid_triangle(side1, side2, side3):
+    if side1 == side2 == side3:
+        print("Треугоьник равносторонний")
+    elif side1 == side2 or side1 == side3 or side2 == side3:
+        print("Треугольник равнобедренный")
+    else: print("Треугольник разносторонний")
+        
+    if ((side1 + side2) > side3) and ((side1 + side3) > side2) and ((side3 + side2) > side1):
+        return True
+    else:
+        return False
```

Comment:
Мне не совсем понятно, зачем эта конструкция, если выше уже есть условия, которые возвращают тип треугольника. Наверно, логичней эти условия располагать выше. Скобки и `else` не нужны
```suggestion
    if side1 + side2 > side3 and side1 + side3 > side2 and side3 + side2 > side1:
        return True

    return False
```

Reply:
Вот о чем я и говорил выше 
![image](https://github.com/LutovinovKN/Immersion_in_Python/assets/49741194/26d76577-1e43-494f-bb1c-7754546933d2)
![image](https://github.com/LutovinovKN/Immersion_in_Python/assets/49741194/1d8c6dd8-c894-4f36-931b-24988c3c8450)